docs/Routing.md

```diff
@@ -0,0 +1,86 @@
+# Routing
+
+[<< Back to index](./index.md)
+
+1. [Basic Routing Principles](#basic-routing-principles)
+2. [FirstSpirit Navigation Service](#firstspirit-navigation-service)
+3. [The triggerRouteChange Function](#the-triggerroutechange-function)
+
+## Basic Routing Principles
```

Comment:
This topic should come after `FirstSpirit Navigation Service`. 
Normally it is just to a way to extend or override the existing paths, that come from the navigation service. 
One important thing to add, is that an existing page in the `pages` folder will definitely override the corresponding page that is coming from the navigation service.
And there are a few more options with the naming convention like for dynamic routes. 
You could link the matching page of the nuxt documentation here as well.